articles/tokens/access-token.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Access Token
+description: This page explains an overview about Auth0 access tokens.
+---
+
+# Access Token
+
+## Overview
+
+The Access Token, commonly referred to as `access_token` in code samples, is a credential that can be used by a client to access an API. The `access_token` should be used as a `Bearer` credential and transmitted in an HTTP `Authorization` header to the API. Auth0 uses access tokens to protect access to the Auth0 Management API.
+
+By default, Auth0 generates access tokens in JSON Web Token (JWT) format, an industry standard. JWTs contain three parts: a header, a set of claims, and a signature:
+- The header contains metadata about the type of token and the cryptographic algorithms used to secure its contents.
+- The set of claims contains verifiable security statements such as the identity of the user and the permissions they are allowed.
+- The signature is used to validate that the token is trustworthy and has not been tampered with.
+
+It should be noted that other token formats are also available. For example, Auth0 Management API v1 (which has been deprecated) uses an opaque token format in which claims are referenced in a separate database, rather than directly in the token. Simple Web Token is an example of another token format, although it has not seen widespread adoption. Support for future tokens formats will be implemented if and when industry consensus develops around them.
```

Comment:
"Other token formats are available" seems to imply that it's possible to configure Auth0 APIs/clients to return tokens in a different format. Since this is not the case as of today, I would reword this paragraph.

The point that we need to try to get across is that access tokens are opaque to clients.